operator/controllers/mlops/experiment_controller.go

```diff
@@ -82,10 +83,15 @@ func (r *ExperimentReconciler) handleFinalizer(ctx context.Context, logger logr.
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.10.0/pkg/reconcile
 func (r *ExperimentReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	logger := log.FromContext(ctx).WithName("Reconcile")
+	logger := log.FromContext(ctx).WithName("ExperimentReconcile")
 	ctx, cancel := context.WithTimeout(ctx, constants.ReconcileTimeout)
 	defer cancel()
 
+	now := time.Now()
+	defer func() {
+		logger.Info("Finished Experiment Reconcile", "duration", time.Since(now))
```

Comment:
To simplify debugging (and matching with scheduler logs), should we add the name of the k8s object being reconciled (experiment, model, etc) as well?